src/openms/source/SIMULATION/LABELING/BaseLabeler.cpp

```diff
@@ -86,25 +86,23 @@ namespace OpenMS
       {
         continue;
       }
-      for (std::vector<ProteinHit>::const_iterator protein_hit = (mapit).getProteinIdentifications()[0].getHits().begin();
-           protein_hit != (mapit).getProteinIdentifications()[0].getHits().end();
-           ++protein_hit)
+      for (const ProteinHit& protein_hit : (mapit).getProteinIdentifications()[0].getHits())
       {
-        if (prot_hits.count((*protein_hit).getSequence())) // we already know this protein -- sum up abundances
+        if (prot_hits.count((protein_hit).getSequence())) // we already know this protein -- sum up abundances
         {
-          SimTypes::SimIntensityType new_intensity = prot_hits[(*protein_hit).getSequence()].getMetaValue("intensity");
+          SimTypes::SimIntensityType new_intensity = prot_hits[(protein_hit).getSequence()].getMetaValue("intensity");
 
           // remember channel intensity
-          prot_hits[(*protein_hit).getSequence()].setMetaValue("intensity_" + String(channel_index), new_intensity);
+          prot_hits[(protein_hit).getSequence()].setMetaValue("intensity_" + String(channel_index), new_intensity);
```

Comment:
there seem to be still some of the unneded brackets
e.g., no need to write (protein_hit).getSequence() instead of
protein_hit.getSequence()

Reply:
can you check your PR one last time (e.g., here https://github.com/OpenMS/OpenMS/pull/5493/files )
to see if there are other cases?

Reply:
okay, no problem :)